packages/x-ray/plugin-ios-screenshots/src/plugin.ts

```diff
@@ -48,11 +49,11 @@ export const iOsScreenshots = (options?: TIosScreenshotsOptions): TPlugin<Uint8A
     const closeIosApp = await runIosApp({
       appName: 'X-Ray',
       appId: 'org.nextools.x-ray',
-      iPhoneModel: '8',
-      iOSVersion: '13',
+      iPhoneModel: opts?.iPhoneVersion.toString() ?? '8',
```

Comment:
Actually `iPhoneVersion` should be a string already (it can be `X`, for example), Could you please correct this in the types above and remove `toString()`?

Also here and the line below there is no need for `?` optional chaining operator, `iPhoneVersion` and `iOSVersion` will always be there because of the `opts` + default values spread.

Reply:
Thanks! I see now that `?` is not necessary :) I fixed that to just:
```
      iPhoneModel: opts.iPhoneVersion,
      iOSVersion: opts.iOSVersion,
```
and set the `iOSVersion` in `opts` from `13.2` to `13`, because `13` was the version that was actually used before the change (since value from options wasn't used).
It it ok now?